### ISSUE TITLE
BigQueryStagingTable

### ISSUE THREAD

**Issue by gmcfall** [konigio/konig#1243]
I want a Datasource template named `BigQueryStagingTable` that implements a BigQuery Table with an external datasource provided by a Storage Bucket.

The external Source URI for the table shall have the following form:
```
  gs://${gcpProjectId}-${gcpLandingBucketName}/${resourceType}.${resourceFormat}/*
```

where
-  `gcpLandingBucket` name is the project-wide name of the Cloud Storage landing bucket
- `resourceType` is a short name for the type of resources contained in the bucket.
- `resourceFormat` is the format of the data objects (`csv` or `json`).

The `resourceType` is derived from the base media-type assigned to the Shape 

For example, suppose a shape is assigned the base media type:

```
application/vnd.pearson.edw.staging.product
```
Then the `resourceType` would be...
```
edw.staging.product
```

In short, we take the portion of the media type after the vendor ID.
